src/be/app.py

```diff
@@ -37,7 +41,19 @@ def status():
 
 @app.post("/api/v1/turn_on")
 def turn_on():
+    
     # TODO turn bulb on here
     return "Hello from the server! implement this function"
 
+bulb = None
+
+def discover_bulb():
+    global bulb
+    bulbs = discover_bulbs()
+    bulb_ip = bulbs[0]['ip']
+    bulb = bulb_ip
+
+discover_bulb()
```

Comment:
[nit] <----- when a comment starts with [nit] it just means this comment isn't really that important and you can ignore it if you like, it's just me nit picking

A nice refactor here would be to have the `discover_bulb()` function actually return the bulb object (`yl.Bulb`)

 and then you can have this line just be `bulb = discover_bulb()`, it's cleaner and you won't need to use globals (which can lead to bugs/confusing code)

Reply:
(you'll still need globals to use `bulb` elsewhere though, which is fine)

Reply:
Done